src/components/LayoutTree.tsx

```diff
@@ -721,14 +727,18 @@ const LayoutTree = () => {
                       const simplePathEnd =
                         -(cliff + i) < simplePath.length ? (simplePath.slice(0, cliff + i) as SimplePath) : HOME_PATH
                       const cliffDepth = unroot(pathEnd).length
+
+                      const isRootPath = isRoot(pathEnd)
```

Comment:
We need a general solution that works for all tables, not just tables in the root. 

```
- z
  - x
  - a
    - =view
      - Table
    - =pin
      - true
    - b
      - c
    - d
      - e
```

Reply:
Hmm, I think removing the `isRootPath` conditional would work for a more general solution.